src/Compilers/CSharp/Test/Semantic/Semantics/RefEscapingTests.cs

```diff
@@ -2515,7 +2515,7 @@ public void M(ref Span<int> global)
 }
 public static class Extensions
 {
-    public static void Deconstruct(ref readonly this Span<int> self, out Span<int> x, out Span<int> y) => throw null;
```

Comment:
Thanks. This test actually broke the build because of two concurrent PRs. I'll add the bug to the description for your PR.